runners/spark/spark_runner.gradle

```diff
@@ -25,9 +25,11 @@ applyJavaNature(
   archivesBaseName: (project.hasProperty('archives_base_name') ? archives_base_name : archivesBaseName),
   exportJavadoc: (project.hasProperty('exportJavadoc') ? exportJavadoc : true),
   classesTriggerCheckerBugs: [
+    'Aggregators': 'TODO: file bug',
```

Comment:
Did you mean to leave TODOs here ?

Reply:
Good catch - added a link to where they said they are working on it.